spec/rubocop/cop/style/safe_navigation_spec.rb

```diff
@@ -172,6 +172,81 @@
         end
       end
 
+      context 'if statements' do
+        it 'accepts a single method call inside of a check for the object' do
+          inspect_source(cop, ["if #{variable}",
```

Comment:
It shouldn't have been. I was having trouble with the auto-correct, but I was able to get it working.